### ISSUE TITLE
Better way to install extensions on node bot

### ISSUE THREAD

**Issue by dapersonmgn** [Bibliofile/Blockheads-MessageBot#50]
A better way to install extensions on the node bot would be nice. Instead of having to goto 
"index.cli.ts" and writing a line. >_> It'd be nicer to just drop in a extension folder and have it run.

**Comment by Bibliofile** [comment]
I agree something like this is necessary, but I'm not yet completely sure how it should be done. 

What I've been thinking so far:
- An extensions directory under /config, all directories / files there will be `require()`'d, this makes development easy, and also locking in to a specific extension version if desired. 
- Also load any extensions retrieved with `storage.getObject('extensions', [], false)` from bhfans (only if their ID hasn't been used in the extensions directory) (eval? Indirect eval by writing to a file then `require()`ing?)

**Comment by dapersonmgn** [comment]
I dislike option 2. Seems really weird IMO. Maybe allow installation of extensions from bhfans? (Download extension to extensions folder and run the bot)

**Comment by Bibliofile** [comment]
Coming back to this much later, here's what I plan on doing. Still subject to change until I actually do it.

1. Add an extension manager to the console loader, this will make it possible to install extensions programatically (so the user can install them from the webpage, if the webpage is enabled)
1. Add scripts so that the user can install an extension using commands, `yarn add-extension bibliofile/quests`
